tests/NPSP.robot

```diff
@@ -263,6 +275,25 @@ Create GAU
     Click Modal Button        Save
     #Sleep    2
     [Return]           ${gau_name}    
+
+Run Donations Batch Process
+    Select App Launcher Tab    NPSP Settings
+    Wait For Locator    frame    Nonprofit Success Pack Settings
+    Select Frame With Title    Nonprofit Success Pack Settings
+    Click Link    link=Bulk Data Processes
+    Wait For Locator    link    Rollup Donations Batch
+    Click Link    link=Rollup Donations Batch
+    Click Button With Value    Run Batch
```

Comment:
@skristem  I believe your test is failing here because there are multiple buttons with that exact same value on the page. I ran into this issue this issue with the levels test. From what I can tell, the only way to click a Run Batch button, Edit, or Save button in NPSP Settings is to apply a custom class name to any button that needs to be directly addressed. This is because the Settings pages are all loaded at once, meaning that the larger DOM contains multiple instances of these buttons with a value of "Run Batch", etc. From a dom perspective, they're pretty much identical. A style class is needed to differentiate them to allow for selecting/clicking. You can see an example of this here: https://github.com/SalesforceFoundation/Cumulus/pull/3789/files#diff-7603579af6cc38159fc2fa73e7b35c5d

If we come up with a good naming method for the style classes for the button types, I can apply a class name to each button across all settings pages in a separate PR.

Reply:
It looks like @skristem already wrote a "Click Settings Button" keyword which could be useful here though -- it selects a settings page by its title and then traverses to a button inside that page.

Reply:
Oh. I didn't see that keyword. What I did see is that this keyword ("Click Button With Value") is failing with element not visible when I tested this locally. It's the same issue I had run into previously, which is why I suggested the custom class. If `Click Settings Button` can work though, then I think we're good.

Reply:
@force2b thats weird. I didn't had any issues with that Click Button with value keyword. Could be a timing issue rather than the keyword and it looks like tests have passed too after merge.